Source/WebCore/dom/Document.cpp

```diff
@@ -5657,15 +5659,16 @@ void Document::clearAutofocusCandidates()
 void Document::flushAutofocusCandidates()
 {
     ASSERT(isTopDocument());
-    if (m_isAutofocusProcessed)
+    RefPtr page = protectedPage();
```

Comment:
I think this is supposed to call this->page() instead of protectedPage for some pedantic reason I don't understand.  Not sure though.

Reply:
Pretty sure they're functionally equivalent:

`inline Page* Document::page() const
{
    return m_frame ? m_frame->page() : nullptr;
}

inline RefPtr<Page> Document::protectedPage() const
{
    return page();
}`